src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/Sentence.kt

```diff
@@ -86,32 +90,6 @@ data class Sentence(
       "Top-up Expiry Date (Post Sentence Supervision PSS)\t:\t" +
       "${sentenceCalculation.topUpSupervisionDate?.format(formatter)}\n"
   }
-  fun canMergeConsecutivelyWith(secondSentence: Sentence): Boolean {
```

Comment:
All these methods are unused now that we dont throw the CannotMergeSentencesException